src/main/java/org/jabref/gui/sharelatex/ShareLatexProjectDialogViewModel.java

```diff
@@ -0,0 +1,61 @@
+package org.jabref.gui.sharelatex;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javafx.beans.property.SimpleListProperty;
+import javafx.collections.FXCollections;
+
+import org.jabref.gui.AbstractViewModel;
+import org.jabref.gui.StateManager;
+import org.jabref.logic.sharelatex.ShareLatexManager;
+import org.jabref.logic.sharelatex.events.ShareLatexEntryMessageEvent;
+import org.jabref.model.sharelatex.ShareLatexProject;
+
+import com.google.common.eventbus.Subscribe;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+public class ShareLatexProjectDialogViewModel extends AbstractViewModel {
+
+    private static final Log LOGGER = LogFactory.getLog(ShareLatexProjectDialogViewModel.class);
+
+    private final StateManager stateManager;
+    private final SimpleListProperty<ShareLatexProjectViewModel> projects = new SimpleListProperty<>(
+            FXCollections.observableArrayList());
+
+    public ShareLatexProjectDialogViewModel(StateManager stateManager, ShareLatexManager manager) {
+        this.stateManager = stateManager;
+        manager.registerListener(this);
+    }
+
+    public void addProjects(List<ShareLatexProject> projectsToAdd) {
+        this.projects.clear();
+        this.projects.addAll(projectsToAdd.stream().map(ShareLatexProjectViewModel::new).collect(Collectors.toList()));
+    }
+
+    public SimpleListProperty<ShareLatexProjectViewModel> projectsProperty() {
+        return this.projects;
+    }
+
+    @Subscribe
+    public void listenToSharelatexEntryMessage(ShareLatexEntryMessageEvent event) {
```

Comment:
@koppor Here I write the new content to the file

Reply:
@tobiasdiez  Here I receive the whole remote bibtex db as string and overwrite the current active database.

My future vision would be: Apply delta changes (lines/cols-diff) from remote to local bibtex database. 
But that is still a bit more complicated